instrumentation/github.com/aws/aws-lambda-go/otellambda/README.md

```diff
@@ -16,38 +16,65 @@ See [./example](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/
 
 ## Usage
 
-Create a sample Lambda Go application such as below.
+Create a sample Lambda Go application instrumented by the `otellambda` package such as below.
 
 ```go
 package main
 
 import (
 	"context"
 	"fmt"
+	
 	"github.com/aws/aws-lambda-go/lambda"
+	"go.opentelemetry.io/contrib/instrumentation/github.com/aws/aws-lambda-go/otellambda"
 )
 
-type MyEvent struct {
-	Name string `json:"name"`
-}
-
-func HandleRequest(ctx context.Context, name MyEvent) (string, error) {
-	return fmt.Sprintf("Hello %s!", name.Name ), nil
+func HandleRequest(ctx context.Context) (error) {
+	fmt.Println("Hello World!" )
+	return nil
 }
 
 func main() {
-	lambda.Start(HandleRequest)
+	ctx := context.Background()
+	lambda.Start(otellambda.InstrumentHandler(HandleRequest(ctx)))
```

Comment:
I don't think this change is correct.  The previous example was correct and this change makes it likely to not even compile since it reduces to:

```go
lambda.Start(otellambda.InstrumentHandler(nil))
```

Also, the prior state was trying to illustrate what a starting point for an uninstrumented function would look like.